tests/unittest/_torch/visual_gen/test_quant_ops.py

```
@@ -115,6 +116,43 @@ def test_fp8_blockwise_zero_weight(self):
+            with self.subTest(shape=shape):
+                weight = torch.randn(shape, dtype=torch.bfloat16, device="cuda")
+                qweight, scale = quantize_fp8_rowwise(weight)
+
+                self.assertEqual(qweight.dtype, torch.float8_e4m3fn)
+                self.assertEqual(qweight.shape, weight.shape)
+                self.assertEqual(scale.shape, (shape[0],))
+
+    def test_fp8_rowwise_zero_weight(self):
+        """Test FP8 rowwise quantization with zero weight."""
+        weight = torch.zeros(128, 256, dtype=torch.bfloat16, device="cuda")
+        qweight, scale = quantize_fp8_rowwise(weight)
```

Ruff (0.16.0)

[warning] Unpacked variable `scale` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)